packages/SwingSet/docs/virtual-objects.md

```diff
@@ -78,7 +88,7 @@ This defines a simple counter object with two properties in its state, a count a
   console.log(`${barCounter.getName()} count is ${barCounter.getCount()`); // "new bar count is 1"
 ```
 
-Suppose you instead wanted to provide the the increment and decrement capabilities as independent facets.  A simplified version of the above (without the name property, counter registry, and `reset` method) might look like:
+Suppose you instead wanted to provide a durable version with the increment and decrement capabilities made available as independent facets.  A simplified version of the above (without the name property, counter registry, and `reset` method) might look like:
```

Comment:
I think it is confusing to make both of these changes in one transition. How about you first make it multi-faceted and then make it durable? Or the other way around? Or as two separate changes to the original? I know it seems more verbose but I think it's worth it.

Reply:
Actually, as I went to edit this, it became clear that making the example durable only changes the last two lines of the example, so the durable case becomes an easy addendum rather than a whole 'nother example.